src/java.security.jgss/share/classes/sun/security/krb5/internal/KDCRep.java

```diff
@@ -136,6 +137,14 @@ protected void init(DerValue encoding, int req_type)
                         "encoding tag is " +
                         encoding.getTag() +
                         " req type is " + req_type);
+
+                System.out.println(">>> KDCRep: Message in bytes is => ");
+                byte[] dataBytes = encoding.getDataBytes();
+                StringBuilder sb = new StringBuilder();
+                for (byte b : dataBytes) {
+                    sb.append(String.format("%02X ", b));
+                }
```

Comment:
How many bytes do we expect here? If it's often more than, say, a hundred, I think we will want to print multiple lines.

Reply:
I can print it in multiple lines. How many bytes per line do you think would be reasonable?

Reply:
16 bytes per line is a common choice (e.g. `hexdump -C` or `od -Ax -tx1`) with each line preceded by the offset.

Reply:
Changed code to print in multiple lines, same as `hexdump -C` would do.